wfe/wfe.go

```diff
@@ -277,7 +277,14 @@ func (wfe *WebFrontEndImpl) relativeDirectory(request *http.Request, directory m
 		if features.Enabled(features.RandomDirectoryEntry) && v == randomDirKeyExplanationLink {
 			continue
 		}
-		relativeDir[k] = wfe.relativeEndpoint(request, v)
+		switch v.(type) {
```

Comment:
I think you can do `switch v := v.(type) {` to avoid having to type assert a second time below.